packages/migrate/test/migrate.test.ts

```diff
@@ -222,6 +222,45 @@ export default class Hello extends Component {
       expect(report.summary[0].basePath).toMatch(project.baseDir);
     });
 
+    test('with service map', async () => {
+      const [inputs, outputs] = prepareInputFiles(project, ['with-mapped-service.gjs']);
+
+      const input: MigrateInput = {
+        basePath: project.baseDir,
+        sourceFiles: inputs,
+        entrypoint: '',
+        reporter,
+      };
+
+      for await (const _ of migrate(input)) {
+        // no ops
+      }
+      const expected = `// @ts-expect-error @rehearsal TODO TS2307: Cannot find module 'services/moo/moo' or its corresponding type declarations.
```

Comment:
This is because we don't have the [declaration here](https://github.com/rehearsal-js/rehearsal-js/pull/1013/files#diff-4b9ecf2ee31ec65529f94a12d5291120d70750645dd279730335bd2a5a334b0eR11) correct?

Reply:
Yeah. Want to cover a case when service is in the map, but TS can't load it.